src/binding-language.js

```diff
@@ -79,19 +79,17 @@ export class TemplatingBindingLanguage extends BindingLanguage {
   }
 
   parseContent(resources, attrName, attrValue){
-    var parts = attrValue.split(this.interpolationRegex), i, ii;
-    if (parts.length <= 1) { //no expression found
+    var parts = [];
+    parse(attrValue, (type, start, end, extra) => {
+      var value = attrValue.substring(start, end) + (extra || '');
+      var expr = type === 'expr' ? this.parser.parse(value) : null;
+      parts.push({type, value, expr});
+    });
+
+    if (exprs.length == 0) { //no expression found
```

Comment:
Wow. That's weird. I figured since all the tests ran, I wouldn't have errors like this ,but it would seems that there is significantly lacking test coverage O_o. Will fix (and another instance of the "same" bug) ASAP.